core/templates/App.ts

```diff
@@ -154,7 +154,9 @@ angular.module('oppia').config([
       'StoryObjectFactory', 'StorySummaryObjectFactory',
       'StoryViewerBackendApiService', 'SubtopicObjectFactory',
       'SubtopicPageContentsObjectFactory', 'SubtopicPageObjectFactory',
-      'SubtopicViewerBackendApiService', 'SuggestionThreadObjectFactory',
+      'SubtopicViewerBackendApiService',
+      'SuggestionModalForCreatorDashboardBackendApiService',
```

Comment:
Adding services is not required here anymore. Can you please remove it?

Reply:
Done.